lib/bpk-component-text-link/src/BpkTextLink-test.android.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import commonTests from './BpkTextLink-test.common';
+
+jest.mock('react-native', () => {
```

Comment:
This can be removed

Reply:
Just the `jest.mock` right? 

Will also make a separate PR after this to remove this from the boilerplate lib if its no longer needed 😄

Reply:
yep, just the mock. It's not necessary to mock the platform anymore, it's based on the file extension. `.android` or `.ios` :)